src/shipit_code_coverage/shipit_code_coverage/report_generators.py

```diff
@@ -1,54 +1,189 @@
 # -*- coding: utf-8 -*-
 import json
 import os
+import signal
+import subprocess
+from datetime import datetime
 
+import pytz
+import requests
+
+from cli_common.log import get_logger
 from shipit_code_coverage import grcov
 
+logger = get_logger(__name__)
+
+
+class ZeroCov(object):
+
+    DATE_FORMAT = '%Y-%m-%d %H:%M:%S %z'
```

Comment:
Maybe we should just store the day and not the rest of the data? On the frontend side, we're probably just interested in the day, but not more